src/filters/Canvas2dFilterBackend.ts

```diff
@@ -31,7 +31,10 @@ export class Canvas2dFilterBackend {
     sourceHeight: number,
     targetCanvas: HTMLCanvasElement,
   ): T2DPipelineState | void {
-    const ctx = targetCanvas.getContext('2d');
+    const ctx = targetCanvas.getContext('2d', {
+      willReadFrequently: true,
+      desynchronized: true,
+    });
```

Comment:
in 2d filtering we don't leverage the gpu, the context can leverage the 'willReadFrequently' optimization.